src/xpcc/architecture/platform/board/stm32f103c8t6_msdb/board.cfg

```diff
@@ -0,0 +1,10 @@
+[build]
+device = stm32f103c8t6
+
+[parameters]
+core.cortex.0.enable_hardfault_handler_led = true
+core.cortex.0.hardfault_handler_led_port = C
+core.cortex.0.hardfault_handler_led_pin = 13
+
+[openocd]
+configfile = board/stm32f103_msdb.cfg
```

Comment:
Hm, good point. I need to implement local refencing of `.cfg` files inside the board folder by adding it to openocds search path.